MAUI/Calendar/accessibility.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: Accessibility in .NET MAUI Calendar control | Syncfusion
```

Comment:
title: Accessibility in .NET MAUI Calendar Control | Syncfusion

Reply:
I have changed the content as above